File: app/src/main/java/com/example/evgeny/setlist_mobile/model/City.java
package com.example.evgeny.setlist_mobile.model;

/**
 * Created by Evgeny on 23.07.2018.
 */

public class City extends BaseModel {
    public String id;
    public String name;
    public String state;
    public String stateCode;
    public Coords coords;
    public Country country;

    public City() {
        coords = new Coords();
        country = new Country();
    }
}

File: app/src/main/java/com/example/evgeny/setlist_mobile/setlists/City.kt
package com.example.evgeny.setlist_mobile.setlists

import androidx.room.Embedded

data class City(
    val id: String,
    val name: String,
    val state: String,
    val stateCode: String?,
    @Embedded(prefix = "coords_") val coords: Coords?,
    @Embedded(prefix = "country_") val country: Country?) : BaseModel() {
}